e2e/project/frontity.settings.ts

```diff
@@ -61,7 +61,10 @@ const settings: Settings<
     name: "wp-source-errors",
     packages: [
       "e2e-wp-source-errors",
-      "@frontity/wp-source",
+      {
+        name: "@frontity/wp-source",
+        state: { source: { url: "https://test.frontity.org/" } },
```

Comment:
I think it's because I changed all the `state.source.api` to `state.source.url` in the frontity settings of the e2e project, but if it's not needed I'll remove it!

Reply:
Done!